lib/ricohapi/mstorage/client.rb

```diff
@@ -55,8 +57,24 @@ def download(media_id)
       # GET /media/{id}/meta, GET /media/{id}/meta/exif, GET /media/{id}/meta/gpano,
       # GET /media/{id}/meta/user, GET /media/{id}/meta/user/{key}
```

Comment:
I think the following comment out  is not necessary.
```ruby
        # GET /media/{id}/meta, GET /media/{id}/meta/exif, GET /media/{id}/meta/gpano,
       # GET /media/{id}/meta/user, GET /media/{id}/meta/user/{key}
```
Because the same comment was added 62th line, 67th line and 72th line.